packages/credential-governance/src/types/domain-definitions.ts

```diff
@@ -25,6 +25,8 @@ export interface IRoleDefinitionV2 extends IRoleDefinitionText {
   issuer: IIssuerDefinition;
   revoker: IRevokerDefinition;
   enrolmentPreconditions: { type: PreconditionType; conditions: string[] }[];
+  // Default seconds value after which the role is considered expired, if empty then no expiry
+  defaultExpiration?: number;
```

Comment:
To me `expiration` implies a precise point in time while `validityPeriod` is a period from which an `expiration` can be calculated during issuance
```suggestion
  defaultValidityPeriod?: number;
```